kikit/panelize_ui.py

```diff
@@ -1,4 +1,5 @@
 import click
+from kikit.defs import Layer
```

Comment:
In order to generate the correct option choices from the Layer enum, I had to import it at the top level instead of deferring the import to within a command function for performance reasons. `kikit.defs` is currently minimal, so I'm hoping this is not an issue, but let me know if you'd prefer populating the option choices some other way.